geth-instance-ropsten/files/startscript.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+cd /root/eth-net-intelligence-api/
+/usr/bin/pm2 start app.json
+while true
+do
+if [ ! `pgrep geth` ] ; then
+	geth  --testnet --syncmode "fast"  --rpc --rpccorsdomain "*" --rpcapi admin,db,eth,net,web3,personal --cache=1024  --rpcport 8545 --rpcaddr 0.0.0.0 --rpcvhosts='*' --bootnodes "enode://30b7ab30a01c124a6cceca36863ece12c4f5fa68e3ba9b0b51407ccc002eeed3b3102d20a88f1c1d3c3154e2449317b8ef95090e77b312d5cc39354f86d5d606@52.176.7.10:30303,enode://865a63255b3bb68023b6bffd5095118fcc13e79dcf014fe4e47e065c350c7cc72af2e53eff895f11ba1bbb6a2b33271c1116ee870f266618eadfc2e78aa7349c@52.176.100.77:30303"
```

Comment:
I think adding `admin`, `db`, and `personal` can be a security issues since most of these nodes will be exposed to public